src/api/spec/models/requests_data_table_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.describe RequestsDataTable do
+    let!(:user) { create(:confirmed_user, login: 'moi') }
+    let(:source_project) { create(:project_with_package) }
+    let(:source_package) { source_project.packages.first }
+    let(:target_project) { create(:project_with_package) }
+    let(:target_package) { target_project.packages.first }
+    let!(:request_one) {
+      create(:bs_request_with_submit_action,
+             creator: user,
+             priority: 'critical',
+             source_project: source_project,
+             source_package: source_package,
+             target_project: target_project,
+             target_package: target_package)
+    }
+    let!(:request_two) {
+      create(:bs_request_with_submit_action,
+             created_at: 2.days.ago,
+             creator: user,
+             source_project: source_project,
+             source_package: source_package,
+             target_project: target_project,
+             target_package: target_package)
+    }
+
+    context 'sort direction' do
+      context 'defaults to sort by :created_at :desc' do
+        before do
+          @data = RequestsDataTable.new({ format: :json, length: 10, start: 0 }, user).rows
```

Comment:
Why is `format: :json` a param here? It doesn't ever get used by this class does it?